### ISSUE TITLE
Accessibility: Screen reader cannot give information about installation progress 

### ISSUE THREAD

**Issue by SteveL-MSFT** [wixtoolset/issues#5904]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 3.9.1208.0

* Which version of Visual Studio are you building with (if any)?

> N/A

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> N/A

* Which version of .NET are you building with?

> N/A

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> N/A

* Describe the problem and the steps to reproduce it.

> With Windows Narrator enabled, start a MSI installer and get to the part when installation initiates.

* Describe the behavior you expected and how it differed from the actual behavior.

> when installation initiates you can visually see progress, but narrator is not able to tell a visually impaired user any progress information.

**Comment by barnson** [comment]
There's no documentation to indicate Windows Installer's UI engine supports this.
